apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/ReportedErrorEventSizer.cs

```diff
@@ -16,17 +16,9 @@
 
 namespace Google.Cloud.Diagnostics.Common
 {
-    /// <summary>
-    /// An <see cref="ISizer{T}"/> for <see cref="ReportedErrorEvent"/>s.
-    /// </summary>
-    internal sealed class ReportedErrorEventSizer : ISizer<ReportedErrorEvent>
+    internal static class ReportedErrorEventSizer
```

Comment:
We don't need this and the TraceSizer. We could just have `MessageSizer<T> where T : IMessage`, then you could refer to `MessageSizer<ReportedErrorEvent>.GetSize` etc. It would be nice not to have this "bounce" at all, but I don't think there's a very simple way of creating an open delegate :(